<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_s</class>
 <widget class="QMainWindow" name="User_s">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit1">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A452</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit2">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2020</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>26</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit3">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>26</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="lineEdit4">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="pbt1">
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>信息框</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="7">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>---------------------------------------------------------------------------------</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>----------------------------------------------------------------------------</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="7">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Rounded MT Bold'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="pbt2">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
